packages/launchdarkly.cache/1.0.2/launchdarkly.cache.nuspec

```diff
@@ -0,0 +1,23 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2013/05/nuspec.xsd">
+  <metadata>
+    <id>LaunchDarkly.Cache</id>
+    <version>1.0.2</version>
+    <authors>LaunchDarkly.Cache</authors>
+    <owners>LaunchDarkly.Cache</owners>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <licenseUrl>https://raw.githubusercontent.com/launchdarkly/dotnet-cache/master/LICENSE</licenseUrl>
+    <description>LaunchDarkly .NET Cache Library</description>
+    <copyright>Copyright 2018 Catamorphic, Co.</copyright>
+    <dependencies>
+      <group targetFramework=".NETFramework4.5" />
+      <group targetFramework=".NETStandard1.4">
+        <dependency id="NETStandard.Library" version="1.6.1" exclude="Build,Analyzers" />
+      </group>
+      <group targetFramework=".NETStandard1.6">
+        <dependency id="NETStandard.Library" version="1.6.1" exclude="Build,Analyzers" />
+      </group>
+      <group targetFramework=".NETStandard2.0" />
+    </dependencies>
+  </metadata>
+</package>
```

Comment:
do we need to commit files for lib folder?

Reply:
As before, I don't know why they are there, I expected that git would ignore them.